JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -1607,8 +1611,9 @@ class AppInsightsTests extends TestClass {
                 // Assert
                 Assert.ok(trackStub.called, "Track should be called");
                 var rdd = <Microsoft.ApplicationInsights.Telemetry.RemoteDependencyData>(<any>trackStub.args[0][0]).data.baseData;
-                Assert.equal(url, rdd.name);
-                Assert.equal(name, rdd.commandName);
+                Assert.equal(pathName, rdd.name);
+                Assert.equal(url, rdd.commandName);
```

Comment:
Is this right? The url variable holds the plain url, shouldn't the commandName object have "GET "+url?